Tests/StreamChatTests/APIClient/Endpoints/Payloads/ChannelListPayload_Tests.swift

```diff
@@ -21,8 +21,8 @@ final class ChannelListPayload_Tests: XCTestCase {
     func test_hugeChannelListQuery_save() throws {
         let decodedPayload = createHugeChannelList()
         
-        let databaseContainer = DatabaseContainer_Spy()
         measure {
+            let databaseContainer = DatabaseContainer_Spy()
```

Comment:
This should be outside of the measure, we don't want the initialization of it to add on the time

Reply:
True, but we need the DB to be "clear" so save operation actually takes place. if this line is out of `measure` block, DB already has data after the first iteration, so other iterations only overwrite data, without actually creating DTOs. If we always test in the same manner it shouldn't matter much, since improvements will be visible

Reply:
Got you!